closed/openjdk-tag.gmk

```diff
@@ -1 +1 @@
-OPENJDK_TAG := jdk-16.0.2+3
+OPENJDK_TAG := jdk-16.0.2+7
```

Comment:
It appears the commits were merged from `openj9` branch which doesn't have latest commits like `openj9-staging`, hence a new commit is created for this TAG update instead of include the one (3-->7) at staging branch.
This is probably because `openj9-staging` hasn't been promoted yet.
Do you mind two commits for same change>

Reply:
I don't understand your question/concern. The change to this file in particular came from cherry-picking 01805da15ab051c09d0d2a8ad709a23691604d7e (from the openj9-staging branch). We can't simply merge from the openj9-staging branch because that would include #29 that we specifically don't want.

Reply:
To summarize, this introduces only two new commits:
* a7fc70c08f1ae45613645b9cd95a8c85f02dd554 - which merges 0832854db6c9eb19fb408d1e843c0f30c3f336d9 (tagged jdk-16.0.2+7 & jdk-16.0.2-ga) into the v0.27.0-release branch
* 35bf335cfca63aab1239929817c36054ae1eb46e - which is a cherry-pick of 01805da15ab051c09d0d2a8ad709a23691604d7e

All other changes come by way of commits that are ancestors of jdk-16.0.2+7.

Reply:
No concern about the content.
I think there is commit within `openj9-staging` branch by script changing `jdk-16.0.2+3` to `jdk-16.0.2+7`.
Did you have to modify this tag manually hence create another commit?
If that process is to avoid #29, it is fine.

Reply:
01805da15ab051c09d0d2a8ad709a23691604d7e was produced via an automated script.

Reply:
Thanks clarification.